python_lib/qss_compiler/examples/TestOps.py

```diff
@@ -0,0 +1,144 @@
+# (C) Copyright IBM 2024.
```

Comment:
```suggestion
#!/usr/bin/env python
# coding: utf-8
 
# (C) Copyright IBM 2024.
```

Reply:
Updated in https://github.com/openqasm/qe-compiler/pull/276/commits/9393557041fc7ef5c563c97aaaab4f734a4c9b55